src/lwc/gsChecklist/gsChecklist.html

```diff
@@ -0,0 +1,56 @@
+<template>
```

Comment:
Has any accessibility validation been done on this HTML? Generally we stick with pure SLDS base components and leverage the capabilities they provide to ensure that all of our pages are fully accessibility. I recommend reviewing this page and your components (all of them, not just this one) to make sure that they are fully accessible. This is a requirement in NPSP.
https://www.lightningdesignsystem.com/accessibility/overview/